### ISSUE TITLE
External command to open a given path in a new tab/switch to existing in Marta

### ISSUE THREAD

**Issue by eugenesvk** [marta-file-manager/marta-issues#956]
Let's say I'd like to have a system-global shortcut that would open my `~/Downloads` folder in Marta
It should open Marta if it's not already opened and if it's opened, it should ask Marta to open a new tab in the same window (ideally configurable) or switch to this tab if it already exists
I've tried a simple AppleScript for the latter
```
tell application "Marta"
	activate
	open "Applications"
	-- open ("/path/to/" as POSIX file) -- alternative
	return
end tell
```

But it doesn't switch to an existing tab, it always opens a duplicate

P.S. I've read in these [docs](https://marta.sh/docs/advanced/cli/) about a cli command, but it also opens a duplicate tab instead of going to the one that already exists. It also fails when symlinked, so `marta .` symlink fails (although the link is correct), but `Launcher` when invoked directly works, strange

**Comment by page-down** [comment]
> ... cli command, but it also opens a duplicate tab instead of going to the one that already exists. ...

I have already suggested the relevant changes before, as well as the detailed implementation logic.

- Each window with panes should be checked in order.
- The non-duplicated new folder tab should be opened in the most correct pane.

https://github.com/marta-file-manager/marta-issues/issues/750#issuecomment-887494515

There is a related issue below (when Marta is set as the default GUI program for `file`).

https://github.com/marta-file-manager/marta-issues/issues/879

However you track this to the cli command, so perhaps it should remain open.

**Comment by eugenesvk** [comment]
> However you track this to the cli command, so perhaps it should remain open.

Yeah, unfortunately Marta isn't ready yet to become a default file manager, but maybe if your other issue is fixed, the Launcher cli will also get it?
